modules/client/shared/src/main/scala/lucuma/itc/client/json/JsonOps.scala

```diff
@@ -5,15 +5,14 @@ package lucuma.itc.client.json
 
 import io.circe.Json
 import io.circe.syntax.*
-import lucuma.core.syntax.enumerated.*
 import lucuma.core.syntax.string.*
 import lucuma.core.util.Enumerated
 
 trait JsonOps {
 
   extension [A: Enumerated](a: A)
     def asScreamingJson: Json =
-      a.tag.toScreamingSnakeCase.asJson
+      Enumerated[A].tag(a).toScreamingSnakeCase.asJson
```

Comment:
Is the `syntax` `a.tag` version discouraged?

Reply:
I remove id from core by mistake :/